NppMenuSearch/Forms/FlyingSearchForm.cs

```diff
@@ -117,6 +145,14 @@ private void txtSearch_KeyPress(object sender, KeyPressEventArgs e)
                 suppressKeyPress = false;
                 e.Handled = true;
             }
+            else
+            {
+                if (e.KeyChar == '\x7F') // 0x7F corresponds to Ctrl+BackSpace. Why? Ask Microsoft...

```

Comment:
This if block is not helpful. Better would be to set the `suppressKeyPress = true` inside txtSearch_KeyDown() and have no change here at all.